types/flutterwave-node/tslint.json

```diff
@@ -0,0 +1,3 @@
+{ "extends": "dtslint/dt.json", "rules" : {
+    "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false]]}]
```

Comment:
Normally, this file should only contain only the `extends` property.  Why was `rules` required?